api/go.mod

```diff
@@ -3,33 +3,35 @@ module github.com/banzaicloud/koperator/api
 go 1.19
 
 require (
+	dario.cat/mergo v1.0.0
 	emperror.dev/errors v0.8.1
 	github.com/banzaicloud/istio-client-go v0.0.17
-	github.com/cert-manager/cert-manager v1.11.2
-	github.com/imdario/mergo v0.3.13
+	github.com/cert-manager/cert-manager v1.12.2
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/api v0.27.3
+	k8s.io/apimachinery v0.27.3
+	sigs.k8s.io/controller-runtime v0.15.0
```

Comment:
@pregnor do you think this is going to be an issue since it exceeds our targeted K8s version support range for our next release?